hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/ProcedureExecutor.java

```diff
@@ -2029,7 +2053,7 @@ protected boolean keepAlive(long lastUpdate) {
   private final class WorkerMonitor extends InlineChore {
     public static final String WORKER_MONITOR_INTERVAL_CONF_KEY =
         "hbase.procedure.worker.monitor.interval.msec";
-    private static final int DEFAULT_WORKER_MONITOR_INTERVAL = 5000; // 5sec
+    private static final int DEFAULT_WORKER_MONITOR_INTERVAL = 1000; // 1sec
```

Comment:
This mean we will get logging from procedure in the Master log more frequently?

Reply:
yes